fvm/transactionInvoker.go

```diff
@@ -188,27 +250,16 @@ func (executor *transactionExecutor) execute() error {
 }
 
 func (executor *transactionExecutor) ExecuteTransactionBody() error {
-	meterParams, err := getBodyMeterParameters(
-		executor.ctx,
-		executor.proc,
-		executor.txnState,
-		executor.derivedTxnData)
-	if err != nil {
-		return fmt.Errorf("error gettng meter parameters: %w", err)
-	}
-
-	var beginErr error
-	executor.nestedTxnId, beginErr = executor.txnState.BeginNestedTransactionWithMeterParams(
-		meterParams)
-	if beginErr != nil {
-		return beginErr
-	}
+	executor.txnState.Resume(executor.pausedState)
 
 	var invalidator programsCache.TransactionInvalidator
-	var txError error
-	invalidator, txError = executor.normalExecution()
-	if executor.errs.Collect(txError).CollectedFailure() {
-		return executor.errs.ErrorOrNil()
+	if !executor.errs.CollectedError() {
+
+		var txError error
+		invalidator, txError = executor.normalExecution()
```

Comment:
`txnState.Resume` is just to reset to the paused state, and this is the actual resume of the execution from the paused state, right?
```suggestion
		// resume the execution from the paused state
		invalidator, txError = executor.normalExecution()
```

Reply:
yes